### PR TITLE
Wdt 537 minimize console output

### PR DESCRIPTION
**Pull request by robertpatrick**

### CODE REVIEW

File: core/src/main/java/oracle/weblogic/deploy/logging/StdoutFilter.java [hunk 1]
```diff
@@ -12,16 +12,14 @@
...
         boolean stdOut = true;
-        int level = record.getLevel() == null ? 0 : record.getLevel().intValue();
+        int level = logRecord.getLevel() == null ? 0 : logRecord.getLevel().intValue();
         if (level == Level.WARNING.intValue() || level == Level.SEVERE.intValue()) {
             stdOut = false;
         }
         return stdOut;
     }
-
 }
```
Comment (ddsharpe) [inline core/src/main/java/oracle/weblogic/deploy/logging/StdoutFilter.java:25]: Please add newline to the end of the file.

File: core/src/main/java/oracle/weblogic/deploy/logging/StdoutHandler.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    /**
+     * Override <tt>StreamHandler.close</tt> to do a flush but not
+     * to close the output stream.  That is, we do <b>not</b>
+     * close <tt>System.out</tt>.
+     */
+    @Override
+    public synchronized void close() {
+        flush();
+    }
+}
```
Comment (ddsharpe) [inline core/src/main/java/oracle/weblogic/deploy/logging/StdoutHandler.java:36]: Missing newline.

File: core/src/main/java/oracle/weblogic/deploy/logging/SummaryHandler.java [hunk 3]
```diff
@@ -241,54 +302,15 @@ public String getHead(Handler handler) {
...
-        if (params != null && params.length != 0) {
-            record.setParameters(params);
-        }
-        record.setSourceClassName(CLASS);
-        record.setSourceMethodName("");
-        record.setResourceBundle(LOGGER.getUnderlyingLogger().getResourceBundle());
-        return record;
-    }
-
 }
```
Comment (ddsharpe) [inline core/src/main/java/oracle/weblogic/deploy/logging/SummaryHandler.java:316]: Missing newline.

File: core/src/main/java/oracle/weblogic/deploy/util/WLSDeployExit.java [hunk 4]
```diff
@@ -49,81 +46,13 @@ public static void exit(int error_code) {
...
-        while (handlers.size() > 0 ){
-            Handler bottom = handlers.removeLast();
-            if (!reducedList.contains(bottom)) {
-                reducedList.add(bottom);
-            }
-        }
-        return reducedList;
-    }
-
 }
```
Comment (ddsharpe) [inline core/src/main/java/oracle/weblogic/deploy/util/WLSDeployExit.java:58]: Missing newline.

**Review by rakillen — APPROVED** [review]

**Review by CarolynRountree — APPROVED** [review]

**Pull request merged**